libs/blocks/marketo/marketo.js

```diff
@@ -19,12 +19,32 @@ const FORM_ID = 'form id';
 const BASE_URL = 'marketo host';
 const MUNCHKIN_ID = 'marketo munckin';
 
+const FORM_MAP = {
```

Comment:
Unnecessary extra line
```suggestion
const MUNCHKIN_ID = 'marketo munckin';
const FORM_MAP = {
```